tests/test_other.py

```diff
@@ -11153,8 +11153,10 @@ def test_unistd_fstatfs(self):
 
   # WASMFS tests
 
-  @also_with_wasmfs
-  def test_unistd_dup(self):
```

Comment:
Looks like this reduces test coverage as the older/smaller test also run on the old FS? But that seems ok.

Reply:
This test here I think exists solely for the new FS.   The old FS has its tests under `unistd` .. this tests is under `wasmfs` and even before this change it never ran with the old FS because the unconditional settings WASMFS.     So no change in coverage AFAICT.